app/uploaders/audio_uploader.rb

```diff
@@ -38,7 +38,7 @@ def store_dir
   # Add a white list of extensions which are allowed to be uploaded.
   # For images you might use something like this:
   def extension_whitelist
-    %w(mp3)
+    %w(mp3 mp4)
```

Comment:
@hengsokly why .mp4, are we allowing video recording?

Reply:
@kakada, the file uploading from mobile has mp4 extension.

Reply:
@hengsokly @limkimsan the mobile should not allow the user to create .mp4 as it intensively burns the storage, as well as the streaming(up/down) that will consume lots of performance, please remove .mp4 both on web and mobile